sql_files/queries.sql

```diff
@@ -9,3 +9,78 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+-- TRANSACIONS
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+COMMIT;
+
+
+BEGIN;
+UPDATE animals SET species = 'digimon';
+WHERE name LIKE '%mon';
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'pokemon'
+WHERE species IS NULL OR species = '';
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] Hey @agneta1991 apologies for the confusion but as clearly stated in the project requirements below you should perform a species update for animals that have a name ending in mon. Currently, you have 1 update statement which is this one `UPDATE animals SET species = 'pokemon'
WHERE species IS NULL OR species = '';` so you should include another one where you update the species to pokemon.
- Here's the project requirement as a reminder 
  ![](https://user-images.githubusercontent.com/104968236/268932860-5f8e7d35-5ab4-42bb-8cb4-cd2a92d6f1b3.png)

Reply:
@AbelG101  i have missed on the very first transaction where i have had to set species as unspecified, but the following 2 transactions where there in the code, see the picture below,

![unspecified](https://github.com/agneta1991/vet_clinic/assets/120168752/5f81e944-495b-4adf-8a45-140ff32c680b)


within this code i have updated just the very first transaction which i have missed out on. 
am i still missing something? im so sorry but i think i am more confused now than before 😄

Reply:
with first transaction i have set species from null to unspecified, then rolled back, then i have completed the first bullet point where it says to set species to digimon to all that has mon in the name, as shown on line 22 to 26, and then second bullet point where i set the remaining species of null (that does not have species set yet) to pokemon, as per requirements.

Reply:
https://github.com/agneta1991/vet_clinic/pull/2/files#diff-27da00cfc47e5b4209a9bfc2d2fe3da08323f906e743f12f20590caca22dab64R22-R26

@AbelG101